templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -442,30 +474,31 @@ <h2>
     <section class="p-section">
       <div class="row--50-50">
         <div class="col">
-          <h2>
-            Fast-track compliance
-            <br class="u-hide--small u-hide--medium" />
-            with Ubuntu Pro for Devices
-          </h2>
+          <h2>See Ubuntu Pro in action</h2>
         </div>
         <div class="col">
           <p class="p-section--shallow u-no-padding--bottom">
-            {{ image (
-            url="https://assets.ubuntu.com/v1/c6897007-fast-track-compliance.png",
-            alt="",
-            width="1200",
-            height="800",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-            }}
+            Get long term support and automated vulnerability management for all your devices, and streamline your patchway to compliance with Ubuntu Pro, the proven way to meet major compliance requirements.
           </p>
-          <p class="p-section--shallow u-no-padding--bottom">
-            Canonical offers device manufacturers a convenient subscription to access security maintenance for over 36,000 packages, and harness automation tools for compliance with multiple standards. With Ubuntu Pro for Devices, you can simplify your vulnerability management efforts to comply with the CRA.
+          <hr class="p-rule--muted" />
+          <p>
+            Airlock Digital meets stricter security and compliance requirements with UBuntu Pro – with up to 40% cost savings and performance increases.
+          </p>
+          <p>
+            <a href="/case-study/airlock-digital-ubuntu-pro-azure-case-study"
+               aria-label="Airlock Digital case study">Download the case study</a>
```

Comment:
Very tiny nit-pick but could you add "download" word to aria-label because, if an aria-label attribute is present on a link, screen readers will typically read that text instead of the visible text of the link.